scripts/handle-cnab-package.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+PACKAGING_IMAGE="mcr.microsoft.com/container-package-app:latest"
+
+main() {
+    local repo_root
+    repo_root=$(git rev-parse --show-toplevel)
+
+    local changed
+    changed=$(ct list-changed --config "$repo_root/ct.yaml")
+
+    if [[ -z "$changed" ]]; then
+        exit 0
+    fi
+
+    local num_changed
+    num_changed=$(wc -l <<< "$changed")
+
+    if ((num_changed > 1)); then
+        echo "This PR has changes to multiple charts. Please create individual PRs per chart." >&2
+        exit 1
+    fi
+
+    # Strip charts directory.
+    changed="${changed##*/}"
+
+    if [[ "$CNAB_ACTION" != "VERIFY" && "$CNAB_ACTION" != "RELEASE" ]]; then
+        echo "CNAB_ACTION must be one of: [VERIFY, RELEASE]."
+        exit 1
+    fi
+
+    if [[ "$PR_TITLE" != "[$changed] "* ]]; then
+        echo "PR title must start with '[$changed] '." >&2
+        exit 1
+    fi
+
+    if [[ ! -d "./cnab-config/$changed" ]]; then
+        echo "CNAB bundle is not setup for $changed service, skipping.."
+        exit 0
+    fi
+
+    # Update CNAB manifest.yaml to match Helm Chart version.
+    local chartVersion
+    chartVersion=$(grep "version: " "$PWD/charts/$changed/Chart.yaml" | sed 's|version: ||g')
+    echo "Parsed Helm Chart version: $chartVersion"
+    sed -i "s|<chartVersion>|$chartVersion|g" "$PWD/cnab-config/$changed/manifest.yaml"
+
+    # Parse image details from charts/<service>/values.yaml.
+    local imageName
+    local imageTag
+    local imageRegistry
+    local imageDigest
+    local imageLocation
+    # Strip castai- prefix to gen service name.
+    imageName="${changed##*-}"
+    imageTag=$(grep "appVersion: " "$PWD/charts/$changed/Chart.yaml" | sed 's|appVersion: ||g' | sed 's|"||g')
```

Comment:
can we use yq instead of sed hacks?

Reply:
As suggested, where possible,  replaced sed with yq.